client-src/elements/chromedash-enterprise-release-notes-page.ts

```diff
@@ -890,15 +890,53 @@ export class ChromedashEnterpriseReleaseNotesPage extends LitElement {
     s,
     shouldDisplayStageTitleInBold
   ): TemplateResult {
-    // TODO(jrobbins): Implement editing widgets in the next CL.
     const platforms: string[] = s.rollout_platforms;
     const choices = PLATFORM_CATEGORIES;
     const availableOptions = Object.values(choices).filter(
       ([value, label, obsolete]) => !obsolete || platforms.includes('' + value)
     );
 
+    const isPreviewing = this.previewingIds.has(s.id);
+    const editor = html` <sl-textarea
+      class="rollout-details"
+      id="edit-rollout-details-${s.id}"
+      value=${s.rollout_details}
+      size="small"
+      rows="1"
+      resize="auto"
+    >
+    </sl-textarea>`;
+    const preview = html`
+      <div
+        id="preview"
+        style="border:var(--card-border); padding:0 var(--content-padding); min-height:14em; background:var(--table-alternate-background)"
+      >
+        ${autolink(s.rollout_details, [], true)}
+      </div>
+    `;
+    const controls = html`
+      <sl-checkbox class="markdown-checkbox" size="small" checked disabled
+        >Use markdown</sl-checkbox
```

Comment:
Since `chromedash-textarea` is updated to support `alwaysMarkdown`, could you use `<chromedash-textarea>` here instead of `<sl-textarea>`?